deploy/docker/gaiahub.sh

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+TASK=$1
+WHICH=$(which docker)
+SCRIPTPATH=$(pwd -P)
+FILE1="docker-compose-base.yaml"
+FILE2="docker-compose-disk.yaml"
+FILE3="disk.env"
+
+#Prints instructions to show possible commands
+instructions() {
+	echo
+	echo "Usage:"
+	echo " To start the GAIA Hub type: $0 start."
+	echo " To stop the GAIA Hub type: $0 stop."
+	echo " To check if GAIA Hub is running type: $0 status."
+	echo " Simply typing $0 displays this help message."
+	echo
+}
+
+#Checks files I need exist
+check_files_exist() {
+	# If a file I need is missing, inform the user.
+	if ! test -f "$FILE1"; then 
```

Comment:
Strictly speaking, you don't need `test` here.  `if ! [ -f "$FILE1" ]` works just as well.

Reply:
Ok. Changed.